pennylane/qnn/torch.py

```diff
@@ -448,20 +450,17 @@ def construct(self, args, kwargs):
 
     def __getattr__(self, item):
         """If the given attribute does not exist in the class, look for it in the wrapped QNode."""
-        if self._initialized:
-            return getattr(self.qnode, item)
+        with contextlib.suppress(AttributeError):
+            if self._initialized:
+                return getattr(self.qnode, item)
 
-        try:
-            return self.__dict__[item]
-        except KeyError as exc:
-            raise AttributeError(item) from exc
+        return super().__getattr__(item)
 
     def __setattr__(self, item, val):
         """If the given attribute does not exist in the class, try to set it in the wrapped QNode."""
         if self._initialized:
             setattr(self.qnode, item, val)
-        else:
-            self.__dict__[item] = val
+        self.__dict__[item] = val
```

Comment:
does using `super().__setattr__` work here? might be better. tbh this whole thing feels weird to me - how would we know if it should be saved to the qnode or the torch layer? qnode if initialized? always both?

Reply:
I decided to update this so that the qnode is only updated if the property is already on the qnode. Otherwise, we use the super method.